### ISSUE TITLE
Need to better specify CI tests maybe with appveyor.yml file

### ISSUE THREAD

**Issue by dotChris90** [SciSharp/NumSharp#179]
- we need to differ little bit more into "Windows" build and "Linux" build (so multiple CI )
- I am very sure the windows based CI build will be green but now it is red because of Linux Server (check on SciSharp/NumSharp --> CI is not red :(
- This is how ML.NET handle this platform dependencies https://github.com/dotnet/machinelearning
- matrix based check (Windows, Linux, Apple - x64 - debug + release)
- maybe also should add an app veyor file? 
- at moment really have no idea what could be the reason for test fail (just that it is connected to *.so files - but on my ubuntu 18.04 it works ....)

**Comment by dotChris90** [comment]
by the way. is app veyor Linux image x86 or x64?

**Comment by dotChris90** [comment]
already discovered the error ..... 
seems it is my bad. I was thinking before that taking the 2 shared object Libs lapack.so and blas.so is enough for successful build. 

I experimented a little bit more and discovered that appveyor CI works if (and really just if the following command is added as installation command "sudo apt-get -y install libblas-dev liblapack-dev" ..... 
Damn the shared object libs depending still on some other libs which I did not included seems. 

@Oceania2018 is there a possibility to export our configuration as appveyor.yaml file or sth in this way?
